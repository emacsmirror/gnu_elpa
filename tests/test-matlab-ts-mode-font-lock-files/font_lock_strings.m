% -*- matlab-ts -*-

s1 = "abc";
s2 = 'One\nTwo';
s3 = sprintf("see %d:%d", 1, 2)

A = [1 2; 3 4];
B = A';   % transpose
C = A'';  % transpose transpose ==> isequal(C,A)
D = A'''; % transpose transpose transpose ==> isequal(D,B)
