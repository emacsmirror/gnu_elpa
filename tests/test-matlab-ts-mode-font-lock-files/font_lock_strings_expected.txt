c ccc ccccccccc ccc

vv o SsssSD
vv o SsssEEsssSD
vv o BBBBBBBbSsss EEsEESD nD nb

v o bn nD n nbD
v o doD   c ccccccccc
v o dooD  c ccccccccc ccccccccc ccc cccccccccccc
v o doooD c ccccccccc ccccccccc ccccccccc ccc cccccccccccc
